### PR TITLE
PMM-9517 Fix scrape configs for old mongodb_exporters.

### PR DESCRIPTION
**Pull request by BupycHuk**
PMM-9517

Build: SUBMODULES-2312

- [ ] Links to other linked pull requests (optional).

### CODE REVIEW

**Review by artemgavrilov — APPROVED** [review]
LGTM, but CI is red

File: services/victoriametrics/prometheus.go [hunk 1]
```diff
@@ -85,6 +87,7 @@ func AddScrapeConfigs(l *logrus.Entry, cfg *config.Config, q *reform.Querier, s
 				return errors.WithStack(err)
 			}
 			paramsHost = pmmAgentNode.Address
+			paramPMMAgentVersion = version.MustParse(*pmmAgent.Version)
```
[the remark this answers is not in the document]
Comment (BupycHuk) [reply services/victoriametrics/prometheus.go:90]: pmmAgent always has a version, but other agents doesn't. We use the same structure for pmm agent and other agents which I think is a mistake.

File: services/victoriametrics/scrape_configs.go [hunk 2]
```diff
@@ -372,6 +373,18 @@ func scrapeConfigsForMySQLdExporter(s *models.MetricsResolutions, params *scrape
 }
 
 func scrapeConfigsForMongoDBExporter(s *models.MetricsResolutions, params *scrapeConfigParams) ([]*config.ScrapeConfig, error) {
+	if params.pmmAgentVersion == nil || params.pmmAgentVersion.Less(version.MustParse("2.25.99")) {
```
[the remark this answers is not in the document]
Comment (BupycHuk) [reply services/victoriametrics/scrape_configs.go:376]: Agree, but it had to be done quickly, that's why I done it this way.

**Review by ShashankSinha252 — APPROVED** [review]

**Review by JiriCtvrtka — APPROVED** [review]

**Comment by JiriCtvrtka** [comment]
Please take a look on failiing tests.

File: services/victoriametrics/prometheus.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (artemgavrilov) [reply services/victoriametrics/prometheus.go:90]: Ok, got it, thanks 👌

**Pull request merged**